packages/connect/src/data/DataManager.ts

```diff
@@ -1,51 +1,23 @@
 // origin: https://github.com/trezor/connect/blob/develop/src/js/data/DataManager.js
 
-import { httpRequest } from '../utils/assets';
 import { parseCoinsJson } from './coinInfo';
 import { parseFirmware } from './firmwareInfo';
 import { parseBridgeJSON } from './transportInfo';
 import { ConnectSettings, DeviceModelInternal } from '../types';
 
-type AssetCollection = { [key: string]: Record<string, any> };
+// We need to declare those imports explicitly so webpack does not include the whole directories with binaries.
+require('@trezor/connect-common/files/coins.json');
+require('@trezor/connect-common/files/coins-eth.json');
+require('@trezor/connect-common/files/bridge/releases.json');
+require('@trezor/connect-common/files/firmware/t1b1/releases.json');
+require('@trezor/connect-common/files/firmware/t2t1/releases.json');
+require('@trezor/connect-common/files/firmware/t2b1/releases.json');
+require('@trezor/connect-common/files/firmware/t3b1/releases.json');
+require('@trezor/connect-common/files/firmware/t3t1/releases.json');
+require('@trezor/connect-common/files/firmware/t3w1/releases.json');
+require('@trezor/protobuf/messages.json');
```

Comment:
this block is weird. could you please expand the comment? I am not sure I follow

Reply:
hmm I am sure it must be solvable in a better way, this kind of requires shouldn't be needed 🤔

Reply:
We already do something similar in https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/utils/assetUtils.ts#L8

Reply:
But I can try different way to tell Webpack that it only has to include those files.